lib/checkmemoryleak.cpp

```diff
@@ -1006,6 +1006,15 @@ void CheckMemoryLeakNoVar::checkForUnusedReturnValue(const Scope *scope)
         if (tok != tok->next()->astOperand1())
             continue;
 
+        if (getReallocationType(tok, 0) == File) {
```

Comment:
Unsure if this also needs to be (for example) OtherRes as well? Or if I should just ignore this and do it for all checks.

Reply:
I think we can try this.